.gitattributes

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
+* text eol=lf
```

Comment:
I suspect this is too aggressive and will break any images. Maybe limit this to py, rst, txt, yaml and toml?

Reply:
I also added two more to the list you mentioned:
```
.license
.md
```
There are plain text files with both of those extensions checked into the repo as well. I can remove them if they're unneeded in this list though.